setup.cfg

```diff
@@ -0,0 +1,24 @@
+[flake8]
+max_line_length = 120
+ignore =
+    W391  # allow empty line at end of file
+    W503  # break before binary operator - allow either style
+    W504  # break after binary operator - allow either style
```

Comment:
I'm not a fan of the following either:

```
E731  # "Do not assign a labmda expression, use a def"
E741  # Ambiguous variable names (e.g. `i` and `I`)
F812  # List comprehension redefines variable
F821  # Undefined name - creates false positive when dynamically creating variables
```

Reply:
I think only add those if we start getting false positives from them. In the WebLab and cellmlmanip repos at present we don't.